test/support/acceptance.go

```diff
@@ -1,34 +1,125 @@
 package support
 
 import (
+	"encoding/json"
+	"gopkg.in/yaml.v3"
 	"log"
 	"os"
 	"path/filepath"
 	"regexp"
 	"strings"
 )
 
-func GetImageDefinitionsFromJson(jsonData map[string]interface{}) []string {
-	var images []string
-	for _, value := range jsonData {
-		switch v := value.(type) {
-		case map[string]interface{}:
-			images = append(images, GetImageDefinitionsFromJson(v)...)
-		case string:
-			if strings.HasPrefix(v, "quay.io") ||
-				strings.HasPrefix(v, "registry.redhat.io") ||
-				strings.Contains(v, "@sha256:") {
-				images = append(images, v)
-			}
+func GetImageDefinitionsFromSnapshot(jsonData Snapshot) []string {
+	return []string{
+		jsonData.CertificateTransparencyGo.CertificateTransparencyGo,
```

Comment:
`"reflect"` package or better to change the `Snapshot` structure

Reply:
Not use any more - parsing snapshot.json into simple map